### DISCUSSION TITLE
Analyzer to bulk lookup FS against hashlookup.circl.lu

### DISCUSSION THREAD

**Discussion by jaegeral** [google/timesketch#1807] [Ideas]
Based on the idea of @sebraven on Twitter: https://twitter.com/Sebdraven/status/1400744250179006465

DO a analyzer that does bulk lookups, maybe 1k a request to hashlookup.circl.lu

**Comment by kiddinn** [comment]
maybe you should create an issue for this, to track it.

**Reply by jaegeral** [reply]
yeah Idea was to first discuss is in the discussions field on how it could be done before creating yet another feature request issue :-)

**Comment by adulau** [comment]
We did many improvements for hashlookup.circl.lu. We now also provide a Bloom filter with all the entries, this can be easily used in Plaso and timesketch. This would avoid any queries on the public service except if meta-data are required.

For more details: https://github.com/hashlookup/hashlookup-forensic-analyser#bloom-filter
